packages/lit-element/src/lib/updating-element.ts

```diff
@@ -669,37 +560,41 @@ export abstract class UpdatingElement extends HTMLElement {
       // in `update` this should not be possible (or an extreme corner case
       // that we'd like to discover).
       // mark state reflecting
-      this._updateState = this._updateState | STATE_IS_REFLECTING_TO_ATTRIBUTE;
+      this._reflectingProperty = name;
```

Comment:
I get that using name vs. boolean was added to help with re-entrancy of setting a different reflecting property in a user `attributeChangeCallback` or custom setter, but I was thrown off around whether there is a correctness issue given that an inner reflecting property will have nulled out `_reflectingProperty` by the time the outer/initiating property's logic runs.

But I guess this is erring on the side of ensuring that setting a different reflecting property definitely causes reflection to happen for that property, at the expense of causing extra reflection for the initiating property? We don't expect that to cause an issue?

Reply:
I suggest we not worry about this at the moment because it may become moot if the [follow on PR](https://github.com/Polymer/lit-html/pull/1245) that changes reflection rules lands. If we don't decide to do that, we can consider if there's a corner case here.